app/local.json

```diff
@@ -32,7 +32,6 @@
         "products": ["Box.com"],
         "type": "swagger",
         "languages" : [".NET"]
-    }
-
-    ]
+    }],
+	"statuscode": "examples/api-explorer-local-apis-overview.html"
```

Comment:
It is.  This is the base version included with the API explorer component build.  You have a AW specific local.json that you clobber this one with.  This shows the portable feature as it could be used by others if they provide a "statuscode" entry in their own local.json file.